static/app/views/relocation/relocation.tsx

```diff
@@ -46,6 +55,31 @@ function getOrganizationOnboardingSteps(): StepDescriptor[] {
 
 function RelocationOnboarding(props: Props) {
   const organization = useOrganization();
+  const [hasPublicKeyError, setHasError] = useState(false);
+  const [publicKey, setPublicKey] = useState('');
+
+  const api = useApi();
+  const fetchData = useCallback(() => {
+    const endpoint = `/publickeys/relocations/`;
+    return api
+      .requestPromise(endpoint)
+      .then(response => {
+        setPublicKey(response.public_key);
+        setHasError(false);
+      })
+      .catch(_error => {
+        setPublicKey('');
+        setHasError(true);
+      });
+  }, [api]);
+
+  useEffect(() => {
```

Comment:
What happens when a user refreshes the page, do we get a different result for the public key? I think since we're not using sessionStorage here that's an issue

Reply:
May need to store this in the `RelocationOnboardingContext`

Reply:
I've added a TODO to add this functionality later - the feature should work without it for now, and its an optimization to an unlikely user flow (refreshing on this page in particular).